gatsby-ssr.js

```diff
@@ -22,4 +22,13 @@ export const onRenderBody = ({ setPostBodyComponents }) => {
       }}
     />,
   ]);
+
+  const link = React.createElement('link', {
```

Comment:
required to be added manually. I found this way to be used by gatsby folks. I wanted to add it to gatsby-browser first but her, it's ok ?

Reply:
yep 👍